addons/website/models/ir_http.py

```diff
@@ -133,17 +133,12 @@ def _add_dispatch_parameters(cls, func):
             request.website = request.website.with_context(request.context)
 
     @classmethod
+    @ormcache()
```

Comment:
same, `ormcache` should not return a recordset

Reply:
Done